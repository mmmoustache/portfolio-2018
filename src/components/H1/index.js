import styled from "styled-components";
import { convertRemToPixels as rem } from '../../utils/utils';

export default styled.h1`
  font-size: ${rem('120px')};
  letter-spacing: -${rem('9px')};
  line-height: ${rem('120px')};
  margin: 0;
`;
