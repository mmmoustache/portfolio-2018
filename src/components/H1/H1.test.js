import React from 'react';
import renderer from 'react-test-renderer';
import H1 from "../H1";

describe('Component: H1', () => {

  it('Should render successfully', () => {
    const component = renderer.create(
      <H1>Test</H1>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });

});
