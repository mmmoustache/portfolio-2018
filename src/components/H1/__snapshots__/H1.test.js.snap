// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: H1 Should render successfully 1`] = `
<h1
  className="sc-bdVaJa eppzNE"
>
  Test
</h1>
`;
